### PR TITLE
Add Aarohi's Riverside Recommendation file

### LINKED ISSUE #13: Create a file for Riverside Recommendations
# Riverside, CA

- [ ] Create a branch
- [ ] Add a commit
- [ ] Open a Pull Request
- [ ] Have a discussion
- [ ] Seek approval
- [ ] Merge Pull Request

### PR DESCRIPTION
**Pull request by aaro7487**
closes #13

### CODE REVIEW

File: aaro7487-Riverside.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Riverside, CA
+
+### Places to Eat
+- GraPow's
+- Cheesecake factory
+
+### Places to Visit
+- Tyler Mall
```
Comment (gracepark) [inline aaro7487-Riverside.md]: Why's it called Tyler Mall?

**Comment by gracepark** [comment]
✋

File: aaro7487-Riverside.md (hunk 1 above)
Comment (aaro7487) [reply aaro7487-Riverside.md]: :laughing:  No idea :wink:

**Review by gracepark — APPROVED** [review]
Looks good to me. ✨

**Pull request merged**